src/Glpi/Kernel/Kernel.php

```diff
@@ -146,6 +147,41 @@ protected function configureRoutes(RoutingConfigurator $routes): void
         //  Global core controllers
         $routes->import($this->getProjectDir() . '/src/Glpi/Controller', 'attribute');
 
+        // This env var is used in autoload in order to determine
+        //   whether we check if plugin is active when autoloading a class.
+        // Since route configuration happens at Kernel's compile-time,
+        //   we cannot (and must not) rely on database calls that checks for plugins,
+        //   so setting this env var to any true-ish value will make sure no DB call is made.
+        // However, the var has to be unset just after, to make sure autoload behaves as expected.
+        // Check the "src/autoload/legacy-autoloader.php" file for more details.
+        $_ENV['GLPI_BYPASS_PLUGINS_CHECKS_IN_AUTOLOAD'] = 1;
+
+        $plugins = glob($this->getProjectDir() . '/plugins/*');
```

Comment:
I missed this earlier, plugins can also be found in the `marketplace` directory.
There is a `PLUGINS_DIRECTORIES` constant that you can use.

Reply:
Solved 👍